go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/balena-os/librsync-go
+
+require (
+	github.com/balena-os/circbuf v0.0.0-20171122095043-56e73111d0b2
+	github.com/sirupsen/logrus v1.3.0
+	github.com/urfave/cli v1.20.0
+	golang.org/x/crypto v0.0.0-20190122013713-64072686203f
+	golang.org/x/sys v0.0.0-20190122071731-054c452bb702 // indirect
```

Comment:
Where does this dependeny comes from?

Reply:
If you're talking about the once with the `// indirect` comment on them, they're Transitive dependencies

Reply:
Yeah, that's what I'm talking about. Why is that transitive dependency in go.mod but all the other ones (see go.sum) are not?

Reply:
I think this might be due to this:

> If a dependency of your module does not itself have a go.mod (e.g., because the dependency has not yet opted in to modules itself), or if its go.mod file is missing one or more of its dependencies (e.g., because the module author did not run go mod tidy), then the missing transitive dependencies will be added to your module's requirements, along with an // indirect comment to indicate that the dependency is not from a direct import within your module.



https://github.com/golang/go/wiki/Modules#why-does-go-mod-tidy-record-indirect-and-test-dependencies-in-my-gomod